diff --git a/node_modules/@chialab/esbuild-plugin-meta-url/lib/index.js b/node_modules/@chialab/esbuild-plugin-meta-url/lib/index.js
index 7e34edb..9a52cac 100644
--- a/node_modules/@chialab/esbuild-plugin-meta-url/lib/index.js
+++ b/node_modules/@chialab/esbuild-plugin-meta-url/lib/index.js
@@ -158,7 +158,7 @@ export default function({ emit = true } = {}) {
                     }
 
                     promises.push(Promise.resolve().then(async () => {
-                        const { path: resolvedPath } = await build.resolve(value.split('?')[0], {
+                        const { path: resolvedPath } = await build.resolve("./" + value.split('?')[0], {
                             kind: 'dynamic-import',
                             importer: args.path,
                             namespace: 'file',
